wp-post-meta-revisions.php

```diff
@@ -138,14 +142,16 @@ public function _wp_check_revisioned_meta_fields_have_changed( $post_has_changed
 	/**
 	 * Save the revisioned meta fields.
 	 *
+	 * @param int $revision_id The id of the revision to save the meta to.
```

Comment:
```suggestion
	 * @param int $revision_id The ID of the revision to save the meta to.
```

Reply:
for some reason i get an error when i try to commit your suggestions...  feel free to commit them if you can

![image](https://user-images.githubusercontent.com/2676022/62959903-c8149380-bdb6-11e9-8752-39577ba44946.png)